bin/cron/test_access_report_import

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/chat_client'
+
+CENSUS_BUCKET = 'cdo-census'.freeze
+
+def update_object_key(object_key, valid)
+  key_without_extension = object_key.chomp(".test")
+  AWS::S3.rename_object(CENSUS_BUCKET, object_key, key_without_extension + ".#{valid ? 'valid' : 'invalid'}")
+end
+
+def main
+  AWS::S3.find_objects_with_ext(CENSUS_BUCKET, 'test', 'state_cs_offerings').each do |object_key|
+    ChatClient.log("Attempting dry run seed of state cs offering file: #{object_key}")
+    Census::StateCsOffering.dry_run_new_test_file(object_key)
+
+    # update object key to reflect valid file
+    update_object_key(object_key, true)
+  rescue => e
+    ChatClient.log("Dry run seeding of state cs offering file failed: #{object_key}", color: 'red')
+    ChatClient.log(e, color: 'red')
+
+    # update object key to reflect invalid file
+    update_object_key(object_key, false)
+    raise e
```

Comment:
Will this break out of the loop? I'm thinking we would want to continue attempting to dry run the remaining files even if one has an error?